.github/workflows/nodejs.yml

```diff
@@ -5,18 +5,23 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ master ]
+    tags:
+      - '*'
+    branches:
+      - '*'
   pull_request:
-    branches: [ master ]
+    branches:
+      - '*'
 
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
 
     strategy:
       matrix:
         node-version: [10.x, 12.x]
```

Comment:
now that the CI is running for 3 different operating systems, it might be worth just running against a single node version (10.x) to keep the number of CI jobs lower. However, the jobs do run quickly, so might not matter...